src/components/FetchQuote.js

```diff
@@ -28,20 +28,24 @@ export default function DisplayQuote() {
   if (loading) {
     return (
       <section className="quote_display">
```

Comment:
- [ ] In the same light kindly make the layout of your Quote page similar to the wireframe layout. There seems to be a very big margin on the left and right of the page. See screenshots below:
- Your layout
![image](https://github.com/tamoorsaeed22/Math-Magician/assets/62849872/870973ee-ce08-43fd-8ea5-2e54f0203e98)
- Wireframe layout
![image](https://github.com/tamoorsaeed22/Math-Magician/assets/62849872/c29f1f85-74ea-4e57-aa11-01a4d812099f)